### ISSUE TITLE
Check if Language impacts date

### ISSUE THREAD

**Issue by potatoqualitee** [potatoqualitee/kbupdate#167]
i think it does

**Comment by codaamok** [comment]
Which date are you referring to? I know Get-HotFix returns the data in the format of the OS culture so it sometimes returns a blank property for dates it cannot parse for "installed date" / "installed on"

**Comment by potatoqualitee** [comment]
Yes, exactly! I believe I parse some dates sometimes but not all dates are formatted the same
